routes/index.js

```diff
@@ -1,10 +1,13 @@
+
+const chatRoutes = require('./chat')
 const relationships = require("./relationships");
 const postsRoutes = require("./posts");
 
 function constructRoutes(app){
     // Example: app.use('/', <middleware name or router name>); // Mounts router or other middleware at path '/'
     app.use('/posts', postsRoutes);
     app.use('/relationships', relationships);
+    app.use('/relationship', chatRoutes);
```

Comment:
We can't have /relationships as one route and /relationship as another route
We can migrate your URLs to /chats if necessary. or we should migrate your functions to relationships.js in routes!

Data level can be different for both but Route should be exposed properly!
We shouldn't confuse clients with /relationships and /relationship